src/client.js

```diff
@@ -38,44 +41,59 @@ export default class Client {
    * @param {String[]} tags Tags to categorize this request in the Dashboard
    * @returns {Promise} An empty Promise, otherwise throws an Error.
    */
-  click({query, documentId, requestId, tags = []}) {
+  click({ query, documentId, requestId, tags = [] }) {
     const params = {
       query,
       document_id: documentId,
       request_id: requestId,
       tags
-    }
+    };
 
-    return this._requestJSON(`${this.clickPath}.json`, params).then(({ response, json }) => {
-      if (!response.ok) {
-        throw new Error(`[${response.status}]${json.errors ? ' ' + json.errors : ''}`)
+    return this._requestJSON(`${this.clickPath}.json`, params).then(
+      ({ response, json }) => {
+        if (!response.ok) {
+          throw new Error(
+            `[${response.status}]${json.errors ? " " + json.errors : ""}`
+          );
+        }
+        return;
       }
-      return
-    })
+    );
   }
 
   _requestJSON(path, params) {
     return this._request(path, params).then(response => {
-      return response.json().then((json) => {
-        return { response: response, json: json }
-      }).catch(() => {
-        return { response: response, json: {} }
-      })
-    })
+      return response
+        .json()
+        .then(json => {
+          return { response: response, json: json };
+        })
+        .catch(() => {
+          return { response: response, json: {} };
+        });
+    });
   }
 
   _request(path, params) {
     const headers = new Headers({
-      'Authorization': `Bearer ${this.apiKey}`,
-      'Content-Type': 'application/json'
-    })
+      Authorization: `Bearer ${this.apiKey}`,
```

Comment:
No quotes required for Authorization?

Reply:
Nope, not in JavaScript.

Reply:
ha, I had the same question

Reply:
JavaScript is weird. 😆

Reply:
Gross would also be an acceptable answer.